src/Orleans/Telemetry/Consumers/TraceTelemetryConsumer.cs

```diff
@@ -37,12 +36,21 @@ public void TrackTrace(string message, Severity severity)
                 case Severity.Off:
                     return;
             }
-            Trace.Flush();
```

Comment:
The global flush function is there, which is great, but that is not enough. We don't call it after every write. So if we want the output to show faster in trace, we have to also flush after every write.
If we are at it, we also supported auto flush in file writer ( one could configure file writer to auto flush, false by default can could be set to true). Take  a look at the older code before the telemetry change.